src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -51,7 +57,12 @@ public Iterable findAll() {
      * @throws {@link BookNotFoundException} if the book with the given identifier does not exist
      */
     @GetMapping("/{id}")
-    public Book findOne(@PathVariable Long id) {
+    @ApiOperation(value = "Giving an id, return the book", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved book"),
+            @ApiResponse(code = 404, message = "Book Not Found")
+    })
+    public Book findOne(@ApiParam(value = "id to find the book", required = true) @PathVariable Long id) {
```

Comment:
seria mejor renombrar este metodo por findById FindOne no es muy diciente

Reply:
Fixed at https://github.com/wolox-training/yh-java/pull/10/commits/1cf106ce63b22f7150cf98d1ca1cda59ef9acd57 and https://github.com/wolox-training/yh-java/pull/10/commits/103648f0f6e0f0b0ebce3c693efe65c6992b65de commits